Source/WebKit/Platform/cocoa/AssertionCapability.mm

```diff
@@ -39,17 +40,13 @@
     , m_willInvalidateBlock { makeBlockPtr(WTFMove(willInvalidateFunction)) }
     , m_didInvalidateBlock { makeBlockPtr(WTFMove(didInvalidateFunction)) }
 {
-#if USE(EXTENSIONKIT)
-    if ([get_SECapabilityClass() respondsToSelector:@selector(assertionWithDomain:name:environmentIdentifier:willInvalidate:didInvalidate:)])
-        m_capability = [get_SECapabilityClass() assertionWithDomain:m_domain name:m_name environmentIdentifier:m_environmentIdentifier willInvalidate:m_willInvalidateBlock.get() didInvalidate:m_didInvalidateBlock.get()];
-    if ([get_SECapabilityClass() respondsToSelector:@selector(assertionWithDomain:name:environmentIdentifier:)])
-        m_capability = [get_SECapabilityClass() assertionWithDomain:m_domain name:m_name environmentIdentifier:m_environmentIdentifier];
-#endif
-}
-
-RetainPtr<_SECapability> AssertionCapability::platformCapability() const
-{
-    return m_capability;
+    RELEASE_LOG(Process, "Taking assertion %s", m_name.utf8().data());
```

Comment:
You want `%{public}s` or it won't show in sysdiagnoses.

Reply:
Also, you may want to add some context at the beginning of the log line (we usually include the `class::function:` that's doing the logging.